README.md

```diff
@@ -22,7 +22,63 @@ to provide the cryptographic primitives.
 # Build
 
 You'll need to [install the Rust toolchain](https://rustup.rs/) (version 1.64
-or above) and a C compiler (`gcc` and `clang` should both work). 
+or above) and a C compiler (`gcc` and `clang` should both work).
+
+## Cryptography provider
+
+Both rustls and rustls-ffi support choosing a cryptography provider for
+implementing the cryptography required for TLS. By default, both will use
+[`aws-lc-rs`][], but [`*ring*`][] is available as an opt-in choice.
+
+It is **not** presently supported to build with both cryptography providers
+activated, or with neither provider activated.
+
+### Choosing a provider
+
+#### Make
+
+When building with the `Makefile`, or example `Makefile.pkg-config` specify
+a `CRYPTO_PROVIDER` as a makefile variable. E.g.:
+
+* `make` to build with the default (`aws-lc-rs`).
+* `make CRYPTO_PROVIDER=aws-lc-rs` to build with `aws-lc-rs` explicitly.
```

Comment:
hm, this disagrees with the makefile (`aws_lc_rs` vs `aws-lc-rs`), ditto other mentions below.

Reply:
This fell away with the change to `aws-lc-rs`. I've done a pass throughout and I don't think there are any `aws_lc_rs` instances left (except in module paths, e.g. `rustls::crypto::aws_lc_rs`).